src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator.UnitTest/QueryTests.cs

```diff
@@ -40,6 +40,8 @@ public void Initialize()
         [DataRow("=5 / 0", "Expression contains division by zero")]
         [DataRow("10+(8*9)/0+7", "Expression contains division by zero")]
         [DataRow("10+(8*9)/0*7", "Expression contains division by zero")]
+        [DataRow("tan(pi/2)", "tan(pi/2) is undefined for odd values of n.")]
+        [DataRow("tan(3*pi/2)", "tan(n*pi/2) is undefined for odd values of n.")]
```

Comment:
```suggestion
        [DataRow("tan(pi/2)", "tan(n*pi/2) is undefined for odd values of n.")]
        [DataRow("tan(3*pi/2)", "tan(n*pi/2) is undefined for odd values of n.")]
```

Both inputs return the same error message as you have only defined one error case/text. (Or is the first case handle in an other place of the code?)

Reply:
You are right